SpreedlySdk.xcodeproj/project.pbxproj

```diff
@@ -1007,10 +1005,7 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				FRAMEWORK_SEARCH_PATHS = (
-					"$(inherited)",
-					"$(PROJECT_DIR)/Frameworks",
-				);
+				FRAMEWORK_SEARCH_PATHS = "$(inherited)";
```

Comment:
Deleting outdated framework search paths was recommended as a fix for the `ld: warning: directory not found for option '-F...` warning.

Src: https://stackoverflow.com/questions/9458739/ld-warning-directory-not-found-for-option